client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -17,7 +19,7 @@ import { getLatestAtomicTransfer } from 'calypso/state/atomic/transfers/selector
 import { pluginInstallationStateChange } from 'calypso/state/marketplace/purchase-flow/actions';
 import { MARKETPLACE_ASYNC_PROCESS_STATUS } from 'calypso/state/marketplace/types';
 import { fetchSitePlugins } from 'calypso/state/plugins/installed/actions';
```

Comment:
I'm a bit confused here: we are introducing a `react-query` query to fetch plugins, but we're also leaving the Redux one intact. We're also implementing the "me sites" endpoint only in the react query version. Shouldn't we port the rest of the plugins fetching to use `react-query` as well?

Reply:
Good catch this needs to be updated.